### ISSUE TITLE
Menu is not consistent/faulty regarding "esc" and "enter".

### ISSUE THREAD

**Issue by jmoerschbach** [widelands/widelands#4596]
**Singleplayer**
- [x] :
1. Click Singplayer - New game
2. Select a map, Hit "enter"
3. Hit "esc"
4. Hit "enter" - nothing happens

---

- [x] :
1. Click Singplayer - Random game
2. Hit "enter"
3. Popup "error because editor is beta" (sometimes, is OK)
4. Hit "enter" - Popup is gone
5. Hit "esc" - segfault

---

- [x] :
1. Click Singplayer - Random game
2. Hit "enter"
3. Popup "error because editor is beta" (sometimes, is OK)
4. Hit "esc" - Popup is gone and I am back in main menu (NOK, I would expect to stay in the current screen to change some settings as the popup text suggests and try again. The Popup should only have the "OK"-Button only)

---

- [x] :
1. Click Singplayer - New campaign
2. Select a campaing, Hit "enter"
3. Hit "esc"
4. Hit "enter" - nothing happens

**Multiplayer**
- [x] :
1. Click Multiplayer - LAN/Direct IP
2. Click "create game" (Note: there is no "enter" shortcut available, but that is ok since there is no "green" button)
3. Hit "esc" - chat box loses focus but nothing else happens (NOK) - I would expect to get back one screen, hitting "esc" more than once does not help, however, clicking a checkbox and hitting "esc" 2x goes back one screen as expected -> maybe editbox focus issue (-> no, in Metaserver Lobby it works,... )?

---

- [x] :
1. Click Multiplayer - Settings
2. Click into the password editbox
3. Hit "enter" - nothing happens (we have a "green" button here!)

**Editor**
- [x] :
1. Click Editor- Load map
2. Hit "esc" - nothing happens (cancel action should be triggered)

**About Widelands**
- [x] :
1. Click "About Widelands"
2. both "esc" and "enter" do not work

**Version:**
 - OS: Windows 10
 - Widelands Version: master

Re  #4591

**Comment by matthiakl** [comment]
One more:
1. Go to Options -> In-Game -> Progress...
2. Hit "enter" - nothing happens (we have a "green" button here)

(also applies to the keyboard options in #4476)
